public/js/track.js

```diff
@@ -279,6 +285,7 @@ function addPlaylistEventListeners() {
 
     createPlaylistButton.addEventListener("click", async function () {
         let isConfirmPlaylistChecked = confirmPlaylistInput.checked;
+        $("#errorAlert").html("").hide();
```

Comment:
I needed to add hide and html with empty string in order to protect against two scenarios:
- Alert only appears once when failure occurs multiple times (user keeps trying to create playlist despite error)
- Alert doesn't appear if there is no longer an error